app/src/main/java/com/example/rmc_app/ui/theme/Type.kt

```diff
@@ -1,37 +1,125 @@
 package com.example.rmc_app.ui.theme
 
 import androidx.compose.material3.Typography
+import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.text.font.Font
 import androidx.compose.ui.text.font.FontFamily
-import com.example.rmc_app.R
-import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontStyle
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.sp
+import com.example.rmc_app.R
 
+// Initialize fonts
+@Suppress("SpellCheckingInspection")
```

Comment:
Instead of having to use the Suppress annotation on each variable you can disable spellchecking in Android Studio in Preferences - Editor - Inspections.

Reply:
I don't want that 😉